README.md

```diff
@@ -1,33 +1,31 @@
-# [project name]
+# Memory-bias for errors dataset
 
 ## Project Goal
-Replace this text with a pithy description of the goal of the data collection project.
+This dataset will include behavioral and EEG data from the participants with ages between 18-25. Participants exhibit normative variation in social anxiety.
```

Comment:
1) Suggest using the present tense.  It will "age" better over time.  (So, "dataset includes")
2) I think we need a little more info here on what "behavioral" data really means?  What kind of task are they completing?
3) You could combine the two sentences for ease of reading: ...participants aged 18-25 years and with normative variation in social anxiety.